web/src/components/ArrowTooltip.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { ArrowTooltip } from "@/components/ArrowTooltip";
+import * as materialUi from "@mui/material";
+import { createTheme, Icon, ThemeProvider, useMediaQuery } from "@mui/material";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+jest.mock("@mui/material", () => mockMaterialUI());
+
+function mockMaterialUI(): typeof materialUi {
+  return {
+    ...jest.requireActual("@mui/material"),
+    useMediaQuery: jest.fn(),
+  };
+}
+
+const isMobile = (value: boolean): void => {
+  (useMediaQuery as jest.Mock).mockImplementation(() => {
+    return value;
+  });
+};
+
+const testString = "test_string";
+
+const renderToolTip = (): void => {
+  render(
+    <ThemeProvider theme={createTheme()}>
+      <ArrowTooltip title={testString}>
+        <div className="fdr fac font-body-lg text-green" data-testid="tooltip">
+          <Icon>help_outline</Icon>
+        </div>
+      </ArrowTooltip>
+    </ThemeProvider>
+  );
+};
+
+describe("<ArrowTooltip />", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it("shows onClick on mobile", async () => {
+    isMobile(true);
+
+    renderToolTip();
+
+    fireEvent.click(screen.getByTestId("tooltip"));
+    await waitFor(() => {
+      expect(screen.getByText(testString)).toBeInTheDocument();
```

Comment:
I think both this test and the one after it need a "before" case assertion to provide adequate confidence about the cause-and-effect here.

Basically: expect string to not be there, fire click, expect string to be there.

Only having the assertion afterwards doesn't prove that it came from the onclick (this test would still pass if the string is just permanently on the page always)